Applications/Utils/MeshEdit/editMaterialID.cpp

```diff
@@ -34,7 +35,12 @@ int main (int argc, char* argv[])
 	TCLAP::CmdLine cmd("Edit material IDs of mesh elements.", ' ', "0.1");
 	TCLAP::SwitchArg replaceArg("r", "replace", "replace material IDs", false);
 	TCLAP::SwitchArg condenseArg("c", "condense", "condense material IDs", false);
-	cmd.xorAdd(replaceArg, condenseArg);
+	TCLAP::SwitchArg specifyArg("s", "specify", "specify material IDs", false);
```

Comment:
What exactly can I do with this option? Can you add a little bit more explanation, please?